resources/windows/wix/sequences.wxs.erb

```diff
@@ -150,7 +150,9 @@
       <Custom Action='SetFromIniPuppetServerPort' Before='FileCost'>
         INI_PUPPET_SERVER_PORT AND NOT PUPPET_SERVER_PORT
       </Custom>
-
+       <Custom Action='EnableLongPathName' Before='InstallFiles'>
+        ENABLE_LONG_PATHS
```

Comment:
Could we prepend the property name with PUPPET_?

Reply:
I don't think this would be a good idea, since this variable only changes system settings and isn't related to Puppet.